src/disassembler.rs

```diff
@@ -95,32 +95,50 @@ fn ldind_str(name: &str, insn: &ebpf::Insn) -> String {
 }
 
 #[inline]
-fn jmp_imm_str<C: ContextObject>(name: &str, insn: &ebpf::Insn, analysis: &Analysis<C>) -> String {
+fn jmp_imm_str(name: &str, insn: &ebpf::Insn, cfg_nodes: &BTreeMap<usize, CfgNode>) -> String {
     let target_pc = (insn.ptr as isize + insn.off as isize + 1) as usize;
     format!(
         "{} r{}, {}, {}",
         name,
         insn.dst,
         insn.imm,
-        resolve_label(analysis, target_pc)
+        resolve_label(cfg_nodes, target_pc)
     )
 }
 
 #[inline]
-fn jmp_reg_str<C: ContextObject>(name: &str, insn: &ebpf::Insn, analysis: &Analysis<C>) -> String {
+fn jmp_reg_str(name: &str, insn: &ebpf::Insn, cfg_nodes: &BTreeMap<usize, CfgNode>) -> String {
     let target_pc = (insn.ptr as isize + insn.off as isize + 1) as usize;
     format!(
         "{} r{}, r{}, {}",
         name,
         insn.dst,
         insn.src,
-        resolve_label(analysis, target_pc)
+        resolve_label(cfg_nodes, target_pc)
+    )
+}
+
+/// Disassemble an eBPF instruction
+pub fn disassemble_instruction<C: ContextObject>(
```

Comment:
Remove this method, and inline it at call sites.

Reply:
Done.